mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -196,6 +197,14 @@ export function PaybackLoanScreen ({
       .catch(logger.error)
   }, [])
 
+  useEffect(() => {
+    if (new BigNumber(amountToPay).lte(0) || amountToPay === '') {
+      setZeroInputAmt(false)
```

Comment:
Naming logic is kinda counter-intuitive here, `zeroInputAmt` implies that the input amount is zero, but the condition here says otherwise

Also can `zeroInputAmt` be changed into another name so that we are able to instantly differentiate that it is a boolean?